bitnami/kafka/Chart.yaml

```
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 12.2.1
+version: 12.2.2
```

```suggestion
version: 12.2.4
```

There has been a couple of PRs on this chart on the last 24h and the current released version is `12.2.3`.

---

@FraPazGal Thanks for your review. Incremented Kafka Helm Chart version to `12.2.4`